### DISCUSSION TITLE
o-my-posh on Linux Powershell

### DISCUSSION THREAD

**Discussion by digitalsigi** [JanDeDobbeleer/oh-my-posh#3105] [General]
Hi,
first, thats a great piece of software! I first installed it on Windows 10 and the on ubuntu with zsh.

Now I'd to have it in powershell unter linux. How is this possible?

**Comment by JanDeDobbeleer** [comment]
@digitalsigi same as Ubuntu. Have a look [here](https://ohmyposh.dev/docs/installation/linux).
